### PR TITLE
Spelling Mistake

### LINKED ISSUE #8080: Manhattan Project Typo
Drink description:
A scienitst drink of choice, for thinking how to blow up the station.

should be 
A scientist's drink of choice, for thinking of how to blow up the station.

### PR DESCRIPTION
**Pull request by Birdtalon**
Fixes #8080 

A single word doesn't deserve a changelog.
Additional grammar fix regarding taping one's shoes... Shoes don't make footsteps, people do. This makes a bit more sense in my mind.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol.dm [hunk 1]
```diff
@@ -610,7 +610,7 @@
 	alcohol_perc = 0.4
 	drink_icon = "proj_manhattanglass"
 	drink_name = "Manhattan Project"
-	drink_desc = "A scienitst drink of choice, for thinking how to blow up the station."
+	drink_desc = "A scientist's drink of choice, for thinking how to blow up the station."
```
Comment (SamHPurp) [inline code/modules/reagents/chemistry/reagents/alcohol.dm]: the `description` too, line `607`

Comment (Birdtalon) [reply code/modules/reagents/chemistry/reagents/alcohol.dm]: 🙈

Comment (SamHPurp) [reply code/modules/reagents/chemistry/reagents/alcohol.dm]: thanks

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -181,7 +181,7 @@
 		if((!silence_steps || shoe_sound) && TR.use(4))
 			silence_steps = 1
 			shoe_sound = null
-			to_chat(user, "You tape the soles of [src] to silence their footsteps.")
+			to_chat(user, "You tape the soles of [src] to silence them.")
```
Comment (tigercat2000) [inline code/modules/clothing/shoes/miscellaneous.dm]: I think this would be better as `to silence your footsteps`, to make it clear what this is doing.

Comment (Birdtalon) [reply code/modules/clothing/shoes/miscellaneous.dm]: Funnily enough I originally had it like that, except if they're not on your feet it doesn't really apply. But that's a fringe case.

Comment (SamHPurp) [reply code/modules/clothing/shoes/miscellaneous.dm]: `their` referring to the shoes, not the person.

I think the original or BT's change is better than Tiger's suggestion for the above stated reason.

**Pull request merged**